### PR TITLE
athena audit logs - always pass utc to query

### PR DESCRIPTION
**Pull request by tobiaszheller**
Part of https://github.com/gravitational/teleport.e/issues/894
RFD: https://github.com/gravitational/teleport/blob/master/rfd/0118-scalable-audit-logs.md

Query now enforces UTC time because parquet writer stores files in folders per date and date is UTC there. I believe UTC is not enforced in auth and it's up to client. This way we are sure that athena querier operates on UTC.

### CODE REVIEW

**Review by rosstimothy — COMMENTED** [review]
How do other backends handle this? Should we consistently use UTC everywhere?

**Comment by tobiaszheller** [comment]
> How do other backends handle this? Should we consistently use UTC everywhere?

TLDR: no need to change in other backends, each backend handles it by themself.

dynamo is using unix helpers, so UTC:
https://github.com/gravitational/teleport/blob/be00c9595f87682de2ba752c2bd7b8df18eaa1ae/lib/events/dynamoevents/dynamoevents.go#L1087-L1088
https://github.com/gravitational/teleport/blob/be00c9595f87682de2ba752c2bd7b8df18eaa1ae/lib/events/dynamoevents/dynamoevents.go#L455-L457

Firestore also using unix helpers:
https://github.com/gravitational/teleport/blob/be00c9595f87682de2ba752c2bd7b8df18eaa1ae/lib/events/firestoreevents/firestoreevents.go#L387-L388

And filework works on time.Time struct so it's safe:
https://github.com/gravitational/teleport/blob/be00c9595f87682de2ba752c2bd7b8df18eaa1ae/lib/events/filelog.go#L489-L506

**Review by rosstimothy — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

**Review by strideynet — APPROVED** [review]
